monitoring/src/main/java/com/l2l3/monitoring/repository/EsMonitoringRepository.java

```diff
@@ -0,0 +1,22 @@
+package com.l2l3.monitoring.repository;
+
+import org.springframework.web.client.RestTemplate;
+
+import com.l2l3.monitoring.model.MonitoringData;
+
+public class EsMonitoringRepository implements MonitoringRepository {
+
+    private final String elasticsearchClusterUrl;
+    private final RestTemplate restTemplate;
+
+    public EsMonitoringRepository(RestTemplate restTemplate, String elasticsearchClusterUrl) {
+	this.restTemplate = restTemplate;
+	this.elasticsearchClusterUrl = elasticsearchClusterUrl;
+    }
+
+    @Override
+    public void save(MonitoringData data) {
+	restTemplate.postForObject(elasticsearchClusterUrl, data, MonitoringData.class);
```

Comment:
About error handling. I am thinking about:
1.  Define an AsyncUncaughtExceptionHandler via AsyncConfigurerSupport
2. If the problem is RestClientException, it could fallback to a different implementation of MonitoringRepository, for example ConsoleMonitoringRepository

Or a failover solution(defining a failover ES url) would be a nice error handling strategy too.